### PR TITLE
[Feature] Translated added German language on Readme file

### LINKED ISSUE #57: [FEATURE] Translate the README file to German
It is necessary to translate the [`README.md`](https://github.com/Fire-Oceann/CS-Space#readme) file into German.
[README/README-DE.md](https://github.com/Fire-Oceann/CS-Space/blob/main/README/README-DE.md)

### PR DESCRIPTION
**Pull request by Rancho2002**
## Fixes Issue #57 

Closes #57

## 🚀 Changes proposed

Added German language on Readme file

## 📷 Screenshots
image

image

## Code of Conduct

- [x] By submitting this pull request, I confirm I've read and complied with the [CoC](https://github.com/Fire-Oceann/CS-Space/blob/main/CODE_OF_CONDUCT.md) 🖖

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

---

You can also join our [Discord](https://discord.gg/bYWVWrDkCU) community.

### CODE REVIEW

**Comment by Rancho2002** [comment]
hi @ziarparvaiz pls review and merge now

**Comment by Rancho2002** [comment]
Hi @Anuran12 , merge this PR

**Comment by Anuran12** [comment]
> Hi @Anuran12 , merge this PR

Tag @ziarparvaiz  please

File: README/README-DE.md [hunk 1]
```diff
@@ -24,22 +24,22 @@
...
-<kbd>[<img title="Русский язык" alt="Русский язык" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/ru.svg" width="22">](./README-RU.md)</kbd>
-<kbd>[<img title="Italiano" alt="Italiano" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/it.svg" width="22">](./README-IT.md)</kbd>
+<kbd>[<img title="Türkçe" alt="Türkçe" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/tr.svg" width="22">](README/README-TR.md)</kbd>
+<kbd>[<img title="Française" alt="Française" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/fr.svg" width="22">](README/README-FR.md)</kbd>
+<kbd>[<img title="Deutsch" alt="Deutsch" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/de.svg" width="22">](README/README-DE.md)</kbd>
+<kbd>[<img title="Русский язык" alt="Русский язык" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/ru.svg" width="22">](README/README-RU.md)</kbd>
+<kbd>[<img title="Italiano" alt="Italiano" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/it.svg" width="22">](README/README-IT.md)</kbd>
```
Comment (ziarparvaiz) [inline README/README-DE.md:31]: the file directory is not true pls add `.` on front of README like this `./README/README-TR`

File: README/README-DE.md [hunk 2]
```diff
@@ -24,22 +24,22 @@
...
+>Ein Informatik-Portal für Enthusiasten. Es enthält gut geschriebene, gut durchdachte und gut erklärte Informatik- und Programmierartikel.
 
-## Prerequisites
+## Voraussetzungen
 
-Make sure you have installed all of the following prerequisites on your development machine:
+Stellen Sie sicher, dass Sie alle folgenden Voraussetzungen auf Ihrem Entwicklungscomputer installiert haben:
 
 - Git - [Download & Install Git](https://git-scm.com/downloads). OSX and Linux machines typically have this already installed.
 - Node.js - [Download & Install](https://nodejs.org/en/download/) Node.js and the npm package manager. If you encounter any problems, you can also use this [GitHub Gist](https://gist.github.com/isaacs/579814) to install Node.js.
```
Comment (ziarparvaiz) [inline README/README-DE.md:40]: I think u forget here to translate

**Review by ziarparvaiz — CHANGES_REQUESTED** [review]

File: README/README-DE.md (hunk 1 above)
Comment (Rancho2002) [reply README/README-DE.md:31]: sure, is there anything wrong more? I am fixing right now

File: README/README-DE.md (hunk 2 above)
Comment (ziarparvaiz) [reply README/README-DE.md:40]: and here to be translate

Comment (Rancho2002) [reply README/README-DE.md:40]: Okay, maybe missed. Doing right now

**Review by ziarparvaiz — APPROVED** [review]
Nice thanks for u contribution

**Pull request merged**